### ISSUE TITLE
remove python dependency

### ISSUE THREAD

**Issue by jayunit100** [kubernetes-sigs/kpng#347]
theres a one liner for templating that we do for ebpf ... we probably dont need j2 , for just templating one file ,,,, do we ?   maybe easier to just do a go template thing which would work on any kpng developer environment

```
620     # Setting vars for generate the kpng deployment based on template
 621     kpng_image="${KPNG_IMAGE_TAG_NAME}" \
 622     image_pull_policy="IfNotPresent" \
 623     backend="${E2E_BACKEND}" \
 624     config_map_name="${CONFIG_MAP_NAME}" \
 625     service_account_name="${SERVICE_ACCOUNT_NAME}" \
 626     namespace="${NAMESPACE}" \
 627     e2e_backend_args="${E2E_BACKEND_ARGS}"\
 628
 629
 630 ############################
 631     j2 "${SCRIPT_DIR}"/kpng-deployment-ds.yaml.j2 -o "${artifacts_directory}"/kpng-deployment-ds.yaml
 632     if_error_exit "error generating kpng deployment YAML"
 633
 634     kubectl --context "${k8s_context}" create -f "${artifacts_directory}"/kpng-deployment-ds.yaml 1> /dev/null
 635     if_error_exit "error creating kpng deployment"
 636
 637     kubectl --context "${k8s_context}" --namespace="${NAMESPACE}" rollout status daemonset kpng -w --request-timeout=3m 1> /dev/null
 638     if_error_exit "timeout waiting kpng rollout"
 639
 640     pass_message "Installation of kpng is done.\n"
```

**Comment by ksankeerth** [comment]
Hi @jayunit100 ,
I'm new to kpng. Shall I try to replace j2 with go template? thanks

**Comment by jayunit100** [comment]
Sure thanks!

**Comment by astoycos** [comment]
Fixed by #258
